templates/.vscode/extensions.json.tpl

```diff
@@ -14,6 +14,10 @@
     "laktak.hjson",
     {{- end }}
 
+    {{- range (stencil.GetModuleHook "wizcli") }}
```

Comment:
suggestion: this should not be specific to `wizcli` - any stencil module should be able to add more suggested VSCode extensions (and the PR title should reflect this).